main.go

```diff
@@ -25,7 +26,8 @@ var (
 	paperlessBaseURL  = os.Getenv("PAPERLESS_BASE_URL")
 	paperlessAPIToken = os.Getenv("PAPERLESS_API_TOKEN")
 	openaiAPIKey      = os.Getenv("OPENAI_API_KEY")
-	tagToFilter       = "paperless-gpt"
+	manualTag         = "paperless-gpt"
+	autoTag           = "paperless-gpt-auto"
```

Comment:
Excellent idea to provide an additional tag for auto-processing! Very easy to opt-in :)